### DISCUSSION TITLE
Only one of all sounds of burning is active

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#15158] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Affliction uses the StatusEffect shared by its AfflictionPrefab, which causes Afflictions of the same Prefab to conflict with each other, and the sound is only one of the affected ones.

https://github.com/user-attachments/assets/148741b1-bcd5-4886-b593-29fc01697224

### Reproduction steps

1. give burning affliction to more than one characters

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by jigardi** [comment]
Have my upvote.
I can't confirm it yet, but I think I ran into this problem yesterday.
I created a flare gun capable of setting impacted characters on fire, which works as intended; but it was strangely quiet - this explains why!
